certified-connectors/AtBot Admin/apiProperties.json

```diff
@@ -0,0 +1,34 @@
+{
+  "properties": {
+    "capabilities": [],
+    "connectionParameters": {
+      "token": {
+        "oAuthSettings": {
+          "clientId": "6145b6dc-1ce4-43b1-a4b5-a19b8440096a",
```

Comment:
This can be removed and replaced with a dummy value

Reply:
I've removed and replaced with a dummy value.  Thanks!